src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/flyout/__jest__/flyout.test.js

```diff
@@ -418,8 +418,12 @@ describe('Flyout', () => {
         },
         obj: {
           searchSource: {
-            getOwnField: () => 'MyIndexPattern*',
+            getOwnField: (field) => {
+              if(field === 'index') { return 'MyIndexPattern*';}
+              if(field === 'filter') { return [{ meta: { index: 'filterIndex' } }];}
+            },
           },
+          _serialize: () => { return { references: [{ id: 'MyIndexPattern*' }, { id: 'filterIndex' }] };},
```

Comment:
a bit confusing that `_serialize` returns an object instead of a string. is it okay?

Reply:
This mocks the `saved_object._serialize()` function which returns an object that can be JSON.stringified. https://github.com/rudolf/kibana/blob/fix-42343-resolve-filter-imports/src/legacy/ui/public/saved_objects/saved_object.js#L309